pkg/cri/resource-manager/resource-manager.go

```diff
@@ -289,6 +298,45 @@ func (m *resmgr) loadConfig() error {
 	return nil
 }
 
+// setupConfigSignal sets up a signal handler for reloading forced configuration.
+func (m *resmgr) setupConfigSignal(signame string) error {
+	if signame == "" {
+		return nil
+	}
+
+	m.Info("setting up signal %s to reload forced configuration", signame)
+
+	sig := unix.SignalNum(signame)
+	if int(sig) == 0 {
+		return resmgrError("invalid forced configuration reload signal '%s'", signame)
+	}
+
+	m.signals = make(chan os.Signal, 1)
+	signal.Notify(m.signals, sig)
+
+	go func(signals <-chan os.Signal) {
+		for {
+			select {
+			case _, ok := <-signals:
+				if !ok {
```

Comment:
Do I understand right that this happens when the signal channel is closed?

Reply:
> Do I understand right that this happens when the signal channel is closed?

Yes.